### ISSUE TITLE
Project Registry App Baseline Monitoring / Alerts

### ISSUE THREAD

**Issue by sbathgate** [BCDevOps/developer-experience#1888]
**Describe the issue**
To ensure proper support and response to Platform Services applications issues, baseline monitoring and alerting is necessary within Sysdig. This includes the following:

Saturation Monitoring (alerts > 80%):
- [x] CPU Utilization
- [x] Memory Utilization
- [x] Storage Utilization

**Definition of done**
When the Platform Services application below have at least the Baseline Monitoring/Alerting Configurations setup.
- [ ] OCP 4 Project Registry

**Comment by w8896699** [comment]
Some issue noticed while setting up monitoring, didn't see quota info for this project namespaces.
TODO LIST:
- [ ] Make sure there is infrastructure as code.
- [ ] Add project set resource quota.
- [ ] Add an alert for proper monitoring.

**Comment by w8896699** [comment]
I have added: 
- `CPU Used vs Requested(Utilization)` where it the percentage between sysdig_container_cpu_cores_used and kube_pod_sysdig_resource_requests_cpu_cores. We can learn how much resources the registry is actually using vs how much it requested
- `CPU Used vs Limited(Threshold)` where is the percentage between sysdig_container_cpu_cores_used and kube_pod_sysdig_resource_limits_cpu_cores. We can learn how much more resources is available for the registry application

For Utilization, we would like to achieve as much high as possible. However, its not always achievable 80% or above, but we will try our best.
For Limitations, I have set up an alert so when its 80%, it will send a rc message to registry internal channel to notify us to give it more resources as it almost run out of the resources. we can then give it more resources or do Horizontal auto scale depends on the situation.
